articles/quickstart/webapp/symfony/01-login.md

```diff
@@ -26,7 +26,11 @@ Add HWIOAuthBundle to `composer.json`.
 
 ${snippet(meta.snippets.dependencies)}
 
-and run `composer update`
+and run `composer update`.
+
+::: note
+This sample is using [`curl-client`](htthttps://github.com/php-http/curl-clienthttps://github.com/php-http/curl-clienthttps://github.com/php-http/curl-clientps://github.com/php-http/curl-client) as PHP HTTP client implementation for [`httplug-bundle`](https://github.com/php-http/HttplugBundle), you can use the PHP HTTP [client implementation](http://docs.php-http.org/en/latest/clients.html) you want.
```

Comment:
I'm not sure whether I'm seeing this wrong on my phone but the URLs look merged in? `htthttps://`